server/chat-commands/admin.ts

```diff
@@ -420,6 +420,57 @@ export const commands: Chat.ChatCommands = {
 		`If a [highlight] is specified, only highlights them if they have that term on their highlight list.`,
 	],
 
+	changeprivateuhtml: 'sendprivatehtml',
+	sendprivateuhtml: 'sendprivatehtml',
+	sendprivatehtml(target, room, user, connection, cmd) {
```

Comment:
Anyway this should be `sendprivatehtmlbox` if it's supposed to have the box.

Reply:
Good point